gruntfile.js

```diff
@@ -108,6 +108,30 @@ module.exports = function (grunt) {
         commitMessage: 'chore: update contributors'
       }
     },
+    conventionalChangelog: {
+      release: {
+        options: {
+          changelogOpts: {
+            preset: 'angular'
+          }
+        },
+        src: 'CHANGELOG.md'
+      },
+    },
+    conventionalGithubReleaser: {
+      release: {
+        options: {
+          auth: {
+            type: 'oauth',
+            token: process.env.GH_TOKEN
```

Comment:
This means I have to set `GH_TOKEN` to a token that has write access to the karma repo correct?